geotrigger.js

```diff
@@ -216,13 +217,14 @@
     var response;
     var httpRequest;
 
-    // assume this is a request to getriggers is it doesn't start with (http|https)://
+    // assume this is a request to geotriggers if it doesn't start with (http|https)://
     var geotriggersRequest = !method.match(/^https?:\/\//);
 
     // create the url for the request
     var url = (geotriggersRequest) ? this.geotriggersUrl + method : method;
 
-    if (this.proxy) {
+    // use a proxy if CORS support isn't present, or developer wants to force it (to do something like pass through credentials)
+    if (this.proxy && this.ignoreCORS || !CORS) {
```

Comment:
my first guess had the effect i was going after in both browsers that support CORS and ones that don't:

1. see if a proxy has even be specified, if not, skip second conditional check entirely and just don't use a proxy.
2. is ignoreCORS truthy?  if so, go ahead and use the proxy
3. if ignoreCORS is falsey, evaluate the last statement.  if no CORS support is present, use the proxy.

i think yours is:

1. see if a proxy has even be specified, if not, skip second conditional check entirely and just don't use a proxy.
2. check to see if ignoreCORS is truthy or the browser lacks CORS support simultaneously, if either, use the proxy

all that being said, i'm happy to make the suggested change because your fu is typically stronger than mine.